content/messaging/customer-chat-api/changelog/index.mdx

```diff
@@ -9,6 +9,16 @@ desc: "Changelog of the Customer Chat API"
 
 This document is the record of all the changes in the **Customer Chat API**, Web and RTM, starting from **version 3.0**.
 
+## [v3.3] - Developer Preview
+
+### Changed
```

Comment:
### Chats
- In the **Deactivate Chat** ([Web](/messaging/customer-chat-api/v3.3/#deactivate-chat) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#deactivate-chat)) method, the `chat_id` parameter was renamed to `id`.

### Properties
- In the **Update Chat Properties** ([Web](/messaging/customer-chat-api/v3.3/#update-chat-properties) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#update-chat-properties)) method, the `chat_id` parameter was renamed to `id`.

Reply:
The rule is to start the sentence with the method/push name -> it's easier to scan the text. Let me know if something is unclear.